WalkingChallenge/Assets.xcassets/create-team.imageset/Contents.json

```diff
@@ -2,12 +2,11 @@
   "images" : [
     {
       "idiom" : "universal",
-      "filename" : "splash.png",
+      "filename" : "create-team.png",
```

Comment:
Do you run any sort of optimizations over these like using something like ImageOptim? Can help keep the binary size down

Reply:
I did not - I think that the conversions are also kinda shoddy at best.  My linux machine is currently in a bad state so I can't really convert this.  It should be an easy thing for someone to just redo the conversions post-commit.

Reply:
I've had really good success with this, keeping the image more or less the same but reducing the binary by several MB. I agree though, this is something that can be done all together before shipping the initial release -- fine to defer till then